### PR TITLE
fix(harborcluster): improve the get -o printing columns

### PR DESCRIPTION
**Pull request by wangcanfengxs**
Signed-off-by: wangcanfeng <wangcanfeng@corp.netease.com>

### CODE REVIEW

File: controllers/goharbor/harborcluster/status.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func newStatus(source *goharborv1.HarborCluster) *status {
 		cr:     source,
 		locker: &sync.Mutex{},
 		data: &goharborv1.HarborClusterStatus{
-			Status:     goharborv1.StatusUnknown,
+			Status:     goharborv1.StatusProvisioning,
```
Comment (steven-zou) [inline controllers/goharbor/harborcluster/status.go:53]: It seems you did not touch any overall status calculation logic in this PR, is that correct?

**Review by steven-zou — COMMENTED** [review]
see my comments

File: controllers/goharbor/harborcluster/status.go (hunk 1 above)
Comment (steven-zou) [reply controllers/goharbor/harborcluster/status.go:53]: Offline discussed, it's clear now.

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by heww — APPROVED** [review]

**Pull request merged**